services/wfm/src/main/java/org/openkilda/wfm/topology/event/OfeLinkBolt.java

```diff
@@ -504,6 +504,12 @@ private void handleIslEvent(Tuple tuple, IslInfoData discoveredIsl, String corre
          *  one place.
          */
         if (IslChangeType.DISCOVERED.equals(state)) {
+            if (discoveredIsl.isSelfLooped()) {
```

Comment:
Is needed to remove such looped link from discovery for some time?

Reply:
no